src/docfx/build/context/LinkResolver.cs

```diff
@@ -11,15 +11,15 @@
 
 namespace Microsoft.Docs.Build
 {
-    internal class DependencyResolver
+    internal class LinkResolver
```

Comment:
This also contains `ResolveContent`, which is inclusion. Is `LinkResolver` accurate?

Reply:
The term `Link` is more about input rather than output, `ResolveLink` uses an input link to resolve an output link, in `ResolveContent` uses an input link to resolve the output content. `ResolveXref` uses an input _uid link_ to resolve and output link.